### ISSUE TITLE
UX Redesign Suggestions

### ISSUE THREAD

**Issue by sionicion** [photoprism/photoprism#975]
I feel like the UX of PhotoPrism needs attention, it reminds me of my Nexus 5 from years ago. Everything is so squared, and the colors are dull.

The grid view of the library along with metadata up front and center all feels so impersonal. I'd rather the themes go away entirely, replacing it with one universal design that can go from light to dark. Bring back color at another point in time.

Display photos grouped by days so users can scroll through their memories. The current views should be secondary, they are more useful looking for metadata but I don't want to see exact date and times of every single picture I'm scrolling through. This is an option in several other photo apps/services. This puts me off especially because I am used to scrolling through my pictures day-by-day or fast scrolling to other months or years, now I just have a very plain grid view or a list that feels more like a computer's filesystem.

In general the UX needs a lot of love, I'm evaluating different self-hosted solutions and UX is a huge factor. It seems to me the current UX seems fitted to work on your library more than actually enjoy using it for it's intended purpose. Like why is there is no sidebar or info pane when you are in the photo viewer? The only way to see info about the current picture is to go to Edit? What sense does that make? What if you accidentally edit metadata when you were merely just trying to view facts about a picture?

I'm still exploring PhotoPrism, I understand you guys have limited time and resources, I get it, but I think gathering ideas for a redesign would be good for when the UX does end up on the roadmap.

**Comment by lastzero** [comment]
To be honest, you sound pretty shocked and your feedback doesn't match with what other users tell us. Given our resources and that we provide all this for free, I think we're in a pretty good place. Many cloud services may be free as well, but they don't give you their source and may abuse your data on top of it.

We'd like to have more metadata in the lightbox as well, but we're using an existing library that doesn't support this. It takes time to develop our own solution. Note that we support much more metadata than many other apps including Google Photos.

Going for perfection and never releasing anything doesn't help us or our users. Things will improve one step at a time. We love plain and simple layouts, you obviously don't. You are welcome to develop a UI that fits your personal taste.

Having said that, I agree with some of the issues you mentioned. It seems most of it is covered by existing GitHub issues, see roadmap: https://github.com/photoprism/photoprism/projects/5

**Comment by lastzero** [comment]
Since you mentioned other self-hosted solutions that are better, can you please provide us with a list so that we can compare them and see their UI?

**Comment by sionicion** [comment]
Ah I didn't see #152 until now.. I should've rewrote this along with waiting until after my coffee wore off, it started as a reply to another issue and then I decided to make a separate issue. I tried reformatting it and it became a condescending mess. I fully appreciate PhotoPrism and what it has to offer.

The mockup in #152 looks really good. That timeline view looks fantastic and would make it so much better. One of my other 2 points is redesigning with one universal design that can be toggled between light or dark. The sidebar on the left and the header for example, stripping the background color away entirely, and then just adjusting for plain white or black. Perhaps rounding a lot of the rectangular hover-overs.

The last point was having more metadata in the photo viewer itself. Yeah just like a fact sheet on the side that is read-only with maybe an edit button to redirect to the current Edit page. This wasn't as important to me, just something I noticed.

As far as other solutions I've used, I used Nextcloud but that's pretty glitchy. I wasn't even able to fully use any solution yet because of Google Photos Takeout. I'm having a lot of trouble with that, on a side note did you guys adapt PhotoPrism for the new "Photos from YEAR" takeout structure? I'm trying google-photos-takeout-helper this time on my library before importing to PhotoPrism but it's what's holding me back. A lot of my pictures have the wrong dates. I want to say I liked the design of Nextcloud Photos and LibrePhotos only because of the mainly white flat design and timeline views. Oh LibrePhotos had a sidebar in the viewer.

Sorry if I came off a little harsh, I should've taken more time to make a more constructive post.
